apps/calendar/src/components/events/horizontalEvent.tsx

```diff
@@ -170,7 +170,7 @@ export function HorizontalEvent({
   });
 
   useEffect(() => {
-    if (isDraggableEvent) {
+    if (isNil(resizingWidth) && isNil(movingLeft)) {
```

Comment:
It's better to make this condition a variable.

Let's create a variable above L147(`isDraggableEvent`), then apply other conditions inside this component.

```diff
@@ -16,7 +16,7 @@
 import { DraggingState } from '@src/slices/dnd';
 import { isSameDate } from '@src/time/datetime';
 import { passConditionalProp } from '@src/utils/preact';
-import { isNil } from '@src/utils/type';
+import { isPresent } from '@src/utils/type';
 
 import type { CalendarColor } from '@t/options';
 
@@ -144,8 +144,8 @@
   const eventContainerRef = useRef<HTMLDivElement>(null);
 
   const { isReadOnly, id, calendarId } = uiModel.model;
-  const isDraggableEvent =
-    !isReadOnlyCalendar && !isReadOnly && isNil(resizingWidth) && isNil(movingLeft);
+  const isDraggingGuideEvent = isPresent(resizingWidth) || isPresent(movingLeft);
+  const isDraggableEvent = !isReadOnlyCalendar && !isReadOnly && !isDraggingGuideEvent;
 
   const startDragEvent = (className: string) => {
     setDraggingEventUIModel(uiModel);
@@ -160,8 +160,7 @@
     if (
       draggingState === DraggingState.DRAGGING &&
       draggingEventUIModel?.cid() === uiModel.cid() &&
-      isNil(resizingWidth) &&
-      isNil(movingLeft)
+      !isDraggingGuideEvent
     ) {
       setIsDraggingTarget(true);
     } else {
@@ -170,7 +169,7 @@
   });
 
   useEffect(() => {
-    if (isNil(resizingWidth) && isNil(movingLeft)) {
+    if (isDraggingGuideEvent) {
       eventBus.fire('afterRenderEvent', uiModel.model.toEventObject());
     }
     // This effect is only for the first render.
```

Reply:
It's good, i'll change this code